src/schemathesis/schemas.py

```diff
@@ -103,6 +105,9 @@ def resolver(self) -> jsonschema.RefResolver:
     def endpoints_count(self) -> int:
         return len(list(self.get_all_endpoints()))
 
+    def get_endpoint_by_operationId(self) -> Dict[str, Any]:
```

Comment:
I do think that we should distinguish two things:
- Filtering of endpoints based on `operationId` key;
- API for getting endpoints by `operationId`.

The latter is not needed for filtration, thus I'd avoid having it, filtration can be implemented in the same way as for `method` / `tag` or `endpoint`

Reply:
Probably I misunderstood your intentions here, this method supports other parts of the implementation